projects/npm-tools/packages/npm-scripts/src/scripts/format.js

```diff
@@ -18,9 +18,21 @@ const DEFAULT_OPTIONS = {
 };
 
 /**
- * File extensions that we want Prettier to process.
+ * File extensions that we allow Prettier to process.
  */
-const EXTENSIONS = ['.js', '.json', '.jsp', '.jspf', '.scss', '.ts', '.tsx'];
+const EXTENSIONS = [
+	'.js',
+	'.json',
+	'.jsp',
+	'.jspf',
+	'.md',
+	'.markdown',
+	'.scss',
+	'.ts',
+	'.tsx',
+	'.yml',
+	'.yaml',
```

Comment:
Likewise, we would use `.yml`, but [I hear](https://github.com/liferay/liferay-frontend-projects/issues/92#issuecomment-701199527) there are some `.yaml` out there too.